lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -324,9 +403,15 @@ public JsonObject asJSON() {
 	 * @param callback (err, tokenDetails)
 	 */
 	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
+		/* Spec: RSA10j */
```

Comment:
We need to fix the existing method please, this was a mistake from some time back and should be fixed so we have a consistent API please